src/axolotl/utils/chat_templates.py

```diff
@@ -17,20 +44,67 @@ def chat_templates(user_choice: str):
     Raises:
         ValueError: If the user_choice is not found in the templates.
     """
+    if user_choice == _JINJA_TEMPALTE_CHOICE:
+        if not jinja_template:
+            raise ValueError(
+                f"`jinja_template` cannot be None when `chat_template` choice is {_JINJA_TEMPALTE_CHOICE}"
+            )
+        return jinja_template
+
+    if user_choice == _DEFAULT_TEMPLATE_CHOICE:
+        if not tokenizer:
```

Comment:
Would this ever be None? It is to my understanding that the tokenizer must be loaded before all of this.

Reply:
There were a few call sites that might not pass tokenizer. I'll check if this is no longer the case and get back

Reply:
There were a few call sites that might not pass tokenizer. I'll check if this is no longer the case and get back

Reply:
Oh, I wasn't aware of that. Resolved